### PR TITLE
Preparations for release FLAC 1.4.1

### PR DESCRIPTION
**Pull request by ktmf01**

### CODE REVIEW

File: doc/release_checklist.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+1. Change version number in
+   - /configure.ac
+   - /CMakeLists.txt
+   - /doc/Doxyfile.in
+   - /man/flac.md
+   - /man/metaflac.md
+   - /test/metaflac-test-files/case07-expect.meta
+1. Change version date in
```
Comment (sezero) [inline doc/release_checklist.md:8]: All items are number 1 ? :)

Comment (ktmf01) [reply doc/release_checklist.md:8]: Yes, when doing that in markdown, it renders properly on github. That way one doesn't have to change the numbering on inserting an item. See [here](https://github.com/xiph/flac/blob/923bd32633cd3667ba404cdf3d24e4e515338668/doc/release_checklist.md)

Comment (sezero) [reply doc/release_checklist.md:8]: Huh. Learnt something new..

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,19 @@
...
 This changelog is not exhaustive, review [the git commit log](https://github.com/xiph/flac/commits) for an exhaustive list of changes.
 
-## FLAC 1.4.0
+## FLAC 1.4.1 (22-Sep-2022)
+
+This release only has a few changes. It was triggered by a problem in the 1.4.0 tarball: man pages were empty and api documentation missing
+
+* CMake fixes (Tomasz Kłoczko)
+* Add checks that man pages and api docs end up in tarball
+* Enable installation of prebuild man pages and api docs
```
Comment (c72578) [inline CHANGELOG.md:11]:
```suggestion
* Enable installation of prebuilt man pages and api docs
```

Comment (ktmf01) [reply CHANGELOG.md:11]: Again... Sorry about that

**Pull request merged**